ibm_catalog.json

```diff
@@ -67,18 +85,22 @@
           "architecture": {
             "features": [
               {
-                "title": " Creates and configures an instance of watsonx.data",
-                "description": "This architecture creates and configures an instance of IBM watsonx.data. It accepts or creates a resource group."
+                "title": "Watsonx data instance",
+                "description": "Creates and configures an instance of watsonx.data."
+              },
+              {
+                "title": "Key Rings and Keys",
+                "description": "Creates Key Rings and Keys for either watsonx.data instances."
               }
             ],
             "diagrams": [
               {
                 "diagram": {
-                  "caption": "watsonx.data on IBM Cloud",
+                  "caption": "IBM Watsonx.data architecture",
                   "url": "https://raw.githubusercontent.com/terraform-ibm-modules/terraform-ibm-watsonx-data/main/reference-architecture/deployable-architecture-watsonx-data.svg",
                   "type": "image/svg+xml"
                 },
-                "description": "This architecture supports creating and configuring an instance of watsonx.data."
+                "description": "This architecture supports creating and configuring an instance of IBM watsonx.data resources."
```

Comment:
Need to add more details as discussed already. 
https://github.ibm.com/GoldenEye/issues/issues/13592#issuecomment-112084362